src/common/event-listener/event-listener-opts.js

```diff
@@ -1,21 +1,31 @@
 import { globalScope } from '../util/global-scope';
 
-var supportsPassive = false
+let passiveSupported = false;
+let signalSupported = false;
 try {
-  var opts = Object.defineProperty({}, 'passive', {
-    // eslint-disable-next-line
-    get: function() {
-      supportsPassive = true
+  const options = {
+    get passive() { // this function will be called when the browser attempts to access the passive property
+      passiveSupported = true;
+      return false;
+    },
+    get signal() {
+      signalSupported = true;
+      return false;
```

Comment:
Why are getter methods setting a value?

Reply:
I think I understand this actually. This is more or less a trick to see if a browser supports these options by checking if these properties are accessed when we create an event listener. Pretty cool but I think we need better documentation here.

Reply:
Maybe a link to the MDN article would suffice.